waiter/config-full.edn

```diff
@@ -1040,7 +1040,24 @@
                                  ;; Stale services are detected by looking at the version of the
                                  ;; token used to create them, such services will have their idle timeout
                                  ;; reduced to stale-timeout-mins + fallback-period-secs.
-                                 "stale-timeout-mins" 15}}
+                                 "stale-timeout-mins" 15}
+
+                ;; Configures tokens-watch-maintainer which handles and updates watches
+                ;; for token events
+                :tokens-watch-maintainer {
```

Comment:
Can we nest this as `:watch-configuration :token`? Then we can add additional components inside `:watch-configuration` as we add more watches (e.g. new instances showing up).